src/posix/platform/netif.cpp

```diff
@@ -1456,4 +1457,20 @@ void platformNetifProcess(const fd_set *aReadFdSet, const fd_set *aWriteFdSet, c
     return;
 }
 
+otError otPlatGetNetif(otInstance *aInstance, const char **outNetIfName, unsigned int *outNetIfIndex)
+{
+    OT_UNUSED_VARIABLE(aInstance);
+
+    otError error;
+
+    VerifyOrExit(sTunIndex != 0, error = OT_ERROR_FAILED);
```

Comment:
If netif failed to initialize, the should already died. So is it necessary to check this?

Reply:
It may not be strictly necessary, but it seems safe and relatively low-cost -- if the initialization logic ever changed to NOT die, this code would continue working.  OK to leave?